cost/aws/unused_ip_addresses/README.md

```diff
@@ -2,7 +2,39 @@
 
 ## What it does
 
-This Policy Template scans all IP addresses in the given account and identifies any unused IP addresses. If any are found, an incident report will show the IP addresses, and related information, and an email will be sent to the user-specified email address. If the user approves that the IP addresses should be deleted, the policy will delete the IP addresses. Optionally, the user can specify one or more tags that if found on an IP address will exclude the IP address from the list.
+This Policy Template scans all IP addresses in the given account and identifies any unused IP addresses. If any are found, an incident report will show the IP addresses, and related information, and an email will be sent to the user-specified email address. If the user approves that the IP addresses should be deleted, the policy will delete the IP addresses. Optionally, the user can specify one or more tags that if found on an IP address will exclude the IP address from the list, as well as specify a minimum number of days for an IP address to be unattached before considering it unused.
```

Comment:
If any are found, an incident report will show the IP addresses, and related information, and an email will be sent to the user-specified email address. 

Read's funny to me: 
If any are found, an incident report will show the IP addresses, related information, and an email will be sent to the user-specified email address.